flags/14.0/vtctld.txt

```diff
@@ -87,132 +56,64 @@ Usage of vtctld:
+      --s3_backup_storage_bucket string                                  S3 bucket to use for backups.
+      --s3_backup_storage_root string                                    root prefix for all backup-related object names.
+      --s3_backup_tls_skip_verify_cert                                   skip the 'certificate is valid' check for SSL connections.
+      --schema_change_check_interval duration                            How often the schema change dir is checked for schema changes (deprecated: if passed as a bare integer, the duration will be in seconds). (default 1m0s)
+      --schema_change_controller string                                  Schema change controller is responsible for finding schema changes and responding to schema change events.
+      --schema_change_dir string                                         Directory containing schema changes for all keyspaces. Each keyspace has its own directory, and schema changes are expected to live in '$KEYSPACE/input' dir. (e.g. 'test_keyspace/input/*sql'). Each sql file represents a schema change.
+      --schema_change_replicas_timeout duration                          How long to wait for replicas to receive a schema change. (default 10s)
+      --schema_change_user string                                        The user who schema changes are submitted on behalf of.
       --security_policy string                                           the name of a registered security policy to use for controlling access to URLs - empty means allow all for anyone (built-in policies: deny-all, read-only)
-      --service_map value                                                comma separated list of services to enable (or disable if prefixed with '-') Example: grpc-vtworker
-      --serving_state_grace_period duration                              how long to pause after broadcasting health to vtgate, before enforcing a new serving state
-      --shutdown_grace_period float                                      how long to wait (in seconds) for queries and transactions to complete during graceful shutdown.
+      --service_map strings                                              comma separated list of services to enable (or disable if prefixed with '-') Example: grpc-queryservice
       --sql-max-length-errors int                                        truncate queries in error logs to the given length (default unlimited)
       --sql-max-length-ui int                                            truncate queries in debug UIs to the given length (default 512) (default 512)
-      --srv_topo_cache_refresh duration                                  how frequently to refresh the topology for cached entries (default 1s)
-      --srv_topo_cache_ttl duration                                      how long to use cached entries for topology (default 1s)
-      --srv_topo_timeout duration                                        topo server timeout (default 5s)
```

Comment:
need to revisit whether it is safe to delete these. though I guess if tests are passing it is fine.